src/profile-logic/marker-styles.js

```diff
@@ -172,6 +172,7 @@ const markerStyles: { +[styleName: string]: MarkerStyle } = {
   GCSlice: gcStyle,
   GCMinor: gcStyle,
   CCSlice: gcStyle,
+  CC: ccStyle,
```

Comment:
I'm not sure if the `CC` marker should look like GCMajor/GCMinor/GCSlice/CCSlice or IdleCCSLice/IdleForgetSkippable/ForgetSkippable. In short: is it triggerred by scripts, or is it triggered by Firefox? @hotsphink do you know?

Reply:
I'm not sure about CCSlice either BTW.

Reply:
I'm not entirely sure what the question is, but the CC marker is triggered by Firefox not directly by scripts. IdleCCSlice and IdleForgetSkippable don't exist anymore, I replaced them with a text payload on CCSlice/ForgetSkippable. Is there currently a visual distinction between Idle versions and non-Idle versions? If that's what you're asking, then this might be tricky. CCSlice now matches GCSlice, in that it might be triggered by either the idle scheduler or by a more direct mechanism (which could even be a direct invocation by script, though I think that's only available to tests?)

The CC marker is analogous to GCMajor—as in, a CC marker delineates a longer interval containing a mixture of multiple CCSlices (during which nothing else is happening on that thread) and regular activity in between slices.

Reply:
Currently there's a color difference between: GCMajor / GCSlice / GCMinor / CCSlice on one hand, and IdleCCSlice, IdleForgetSkippable, ForgetSkippable on the other hand. I didn't do this myself but I think the rationale is that the formers are triggered by script (not directly, but as a result from some code) but the latters aren't. I'm not so sure these expectations are fully true. It might be better to split by GC/CC, or just make them all have the same color.

Eager to ear what your thoughts are @hotsphink!

Reply:
I don't think the trigger is all that useful as a distinguishing criterion. I'd probably go with a GC/CC split, though if it's useful to reduce the number of colors, then using the same color would be fine too. At one point, I would have requested that they be different because the problems were getting misrouted (CC bugs going to GC people and vice versa), but I think the markers and categories are now clear enough that things have been going well.